staff/mediacloner/task-API-pro/src/data/taskData.js

```diff
@@ -0,0 +1,102 @@
+
+const _ = require("lodash")
+
+let tasks = [];
+
+let id = 0;
+
+const taskData = {
+
+    insert (text, username){    
+        id += 1      
```

Comment:
you can increment with id++, because the increment is persistent while server is running.